man/makeGapsInclusiveIfStatistic.Rd

```diff
@@ -0,0 +1,20 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
lol, i just keep ignoring this file in my Rstudio.. good for you.

Reply:
I never know what to include in my commits for R, so went for it...